xcode/Subconscious/Shared/Components/Common/Profile/EditProfileSheet.swift

```diff
@@ -116,14 +116,15 @@ struct EditProfileSheetModel: ModelProtocol {
 
 
 struct EditProfileSheet: View {
-    var state: EditProfileSheetModel
-    var send: (EditProfileSheetAction) -> Void
+    var store: ViewStore<EditProfileSheetModel>
+    
+    var state: EditProfileSheetModel { store.state }
+    var send: (EditProfileSheetAction) -> Void { store.send }
```

Comment:
Would prefer to reference these through `store.` It's easier to track intent in the view IMO.